app/src/main/res/drawable/border_login_button.xml

```diff
@@ -0,0 +1,9 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners
+        android:radius="2dp" />
+    <stroke
+        android:width="1dp"
+        android:color="@color/border" />
+    <solid android:color="@android:color/white" />
+</shape>
```

Comment:
:+1: Nice, we should do something like this for the save button on the Edit Profile activity.